bower.json

```diff
@@ -28,6 +28,7 @@
   },
   "devDependencies": {
     "purescript-quickcheck": "^6.1.0",
-    "purescript-spec": "^3.0.0"
+    "purescript-spec": "^4.0.1",
+    "purescript-aff": "^5.1.2"
```

Comment:
This is acceptable as the bower file is a little out of date, but I don't think we need to touch the `bower.json` file when making updates. It will be removed ASAP and is only here in the case we need to push things to Pursuit using it. No one should be using it for local development (they should be using Spago instead). That's why I didn't include bower_components in .gitignore template.

Reply:
The problem here is that the package set I'm using has an updated version of `spec`. So, the code cannot compile unless I use an older package set. I opted to update the dependencies in `bower.json` rather than use an older package set. Since only the testing code was changed, I don't believe this will cause a breaking change release.

Reply:
Ok, that’s fine. Thanks!